plugin/java/src/main/resources/META-INF/jqassistant-rules/java-suppress.xml

```diff
@@ -0,0 +1,59 @@
+<jqassistant-rules xmlns="http://schema.jqassistant.org/rule/v2.7"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://schema.jqassistant.org/rule/v2.7 http://schema.jqassistant.org/rule/jqassistant-rule-v2.7.xsd">
+
+
+        <constraint id="suppress:suppressFieldsMustProvideAReason">
```

Comment:
The results of all constraints should provide acontain a column with the element that has been annotated with `jQASuppress`